pkg/apis/harvesterhci.io/v1beta1/supportbundle.go

```diff
@@ -26,7 +26,7 @@ type SupportBundle struct {
 
 type SupportBundleSpec struct {
 	// +optional
-	IssueURL string `json:"issueURL"`
+	IssueURL string `json:"issueURL,omitempty"`
```

Comment:
@bk201 u may want to add this to the other PR of https://github.com/harvester/harvester/pull/844, as this may generating a different CRD and API file.

Reply:
+1. Convert to draft